src/renderer/screens/exchange/Swap2/Form/FormLoading/FormLoading.js

```diff
@@ -0,0 +1,13 @@
+// @flow
```

Comment:
Why is this not just in index.js? is it a standard that I'm missing?

Reply:
That's a convention that I use to work with, treat components like modules. I use to create a directory to encapsulate components related files and expose only what is supposed to be exposed from the index file which is the entry point of the "module".
For example, suppose that you have a component that is an input that manipulates dates and times, I use to create structure like this.

```sh
InputDateTime/
  InputDateTime.js # The main component
  InputDateTime.test.js # Component's unit tests
  InputDateTime.snap.test.js # Component's snapshot
  InputDateTime.stories.js # Component's story
  index.js #only export main component. If one of the utils files are needed elsewhere, we've to move it to the common utils/ directory
  utils/
    getCurrentDate.js # utils function
    getCurrentDate.test.js # getCurrentDate's unit tests
    getCurrentTime.js # utils function
    getCurrentTime.test.js # getCurrentDate's unit tests
    useNow.js # custom hooks
    useNow.test.js # custom hooks tests
```

That avoid zombies files because if you move for example tests files to a dedicated directory and the same thing for stories when you delete or update the component you've to think about checking other parts of the project structure to remove/update related files. If everything is encapsulated and located in the same directory, you have got a better overview of the components and that will be easy to maintain. Plus, it is easier to localise the lack of tests for example.

--- 

I used that convention because I used to and I find this one powerful. However, I'm not attached to this one and I can move the logic into the index file if you want. 
However, as the team is growing maybe it could be interesting to discuss some conventions in the future, would be cool to have the point of view of everybody and mix them to find the perfect way to structure our projects!

Reply:
I like your reasoning, so keep it as you do and we can discuss with the team for a standard that we all like later on.